source/Random.cpp

```diff
@@ -34,6 +34,11 @@ namespace {
 #endif
 }
 
+static bool normalBMCached = false;
+static double cachedBMNormal = std::numeric_limits<double>::infinity();
+static constexpr double epsilon = std::numeric_limits<double>::epsilon();
+static constexpr double two_pi = 2.0 * 3.1415926535;
```

Comment:
I just figured this out on my own last night (which means I committed it already, it's a good suggestion)?  I'm rather confused about the time stamp on your review in general because it says it's 4 days old and I was working on this and checking for reviews through the day yesterday.

Reply:
I don't submit review comments immediately. GitHub preserves when the content was authored, rather than applying the review submission time to all submitted comments.

Reply:
Ah, didn't think of that.  Going to resolve this though as it has been taken care of.